keps/sig-cloud-provider/0003-testgrid-e2e.md

```diff
@@ -0,0 +1,270 @@
+---
+kep-number: 003
```

Comment:
I believe https://github.com/kubernetes/community/tree/master/keps/sig-cluster-lifecycle is already using 0003.

Reply:
There's open discussion on KEP numbering here: https://github.com/kubernetes/community/issues/2245

Reply:
Will revisit file name and KEP number once that's resolved :)